assets/js/admin/orders.js

```diff
@@ -11,4 +11,94 @@ jQuery( document ).ready( ( $ ) => {
 
 	'use strict';
 
+	const isCommerceOrder = Boolean( wc_facebook_commerce_orders.is_commerce_order );
+
+	const WCFacebookCommerceOrderOperations = {
```

Comment:
Why do we need an object at all?

Reply:
At first I though about exposing it to the `window` scope for later use, but in the same time it's nice to have them grouped for easy to follow and understand ... I don't have strong feeling about it